File: semester_two/algoDat/public/code/vorlesung-13/Simple.java
// Copyright 2017, University of Freiburg,
// Chair of Algorithms and Data Structures.
// Author: Hannah Bast <bast@cs.uni-freiburg.de>.

/**
 * Main function.
 */
public class Simple {

  static int diff(int a, int b) {
    return a - b;
  }

  public static void main(String[] args) {
    int c = 0;
    int a = 137;
    int b = 42;
    int n = Integer.parseInt(args[0]);
    for (int i = 0; i < n; i++) {
      c += diff(a, b);
    }
  }
}

File: semester_two/algoDat/public/code/vorlesung-13/Simple.cpp
// Copyright 2017, University of Freiburg,
// Chair of Algorithms and Data Structures.
// Author: Hannah Bast <bast@cs.uni-freiburg.de>

// A simple program to show some machine code.
int main(int argc, char** argv) {
  int a = 137;
  int b = 42;
  int c = 0;
  for (int i = 0; i < 100; i++) {
    c += a - b;
  }
  return c;
}
